info/templates/info/event_detail.html

```diff
@@ -38,13 +38,15 @@ <h4>Location:</h4>
         <p>{{ event.location.post_code }}</p>
       </div>
       <div>
-        <p>{{ event.contact }}</p>
+        <span><a href="tel:+4733378901">{{ event.contact.phone_number}}</a>
+        <a href="mailto:https://www.google.com">{{ event.contact.email}}</a></span>
```

Comment:
This is not correct.

The href does not reference the phone number or the email, why are they not being displayed to the user?

Reply:
I kept blank href link now,  as I dont know how to invoke python code in it.